tutor/src/screens/course-settings/delete-course-button.js

```diff
@@ -2,11 +2,20 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { observer } from 'mobx-react';
 import { observable, action } from 'mobx';
-import { Modal, Button, OverlayTrigger, Tooltip } from 'react-bootstrap';
+import styled from 'styled-components'
+import { Modal, Button } from 'react-bootstrap';
 import { isEmpty } from 'lodash';
 import { Icon } from 'shared';
+import { colors } from 'theme'
 import Course from '../../models/course';
 
+const StyledModal = styled(Modal)`
+    p.danger {
+        font-weight: 700;
+        color: ${colors.strong_red};
+    }
+`
```

Comment:
In the interests of not duplicating more than needs to be, bootstrap has a `.text-danger` class. How about using that?  it could be re-declared if needed by setting the `$danger` variable in `tutor/resources/styles/variables/tutor-bootstrap.scss`